src/Testcontainers/Configurations/Modules/Databases/CosmosDbTestcontainerConfiguration.cs

```diff
@@ -0,0 +1,67 @@
+namespace DotNet.Testcontainers.Configurations
+{
+    using DotNet.Testcontainers.Builders;
+    using JetBrains.Annotations;
+
+    [PublicAPI]
+    public class CosmosDbTestcontainerConfiguration : TestcontainerDatabaseConfiguration
+    {
+        private const string CosmosDbImage = 
+            "mcr.microsoft.com/cosmosdb/linux/azure-cosmos-emulator";
+
+        private const int CosmosDbPort = 8081;
+
+        private const string CosmosDbDefaultKey =
+            "C2y6yDjf5/R+ob0N8A7Cgv30VRDJIWEHLM+4QDU5DE2nQ9nDuVTqobD4b8mGGyPMbIZnqyMsEcaGQy67XIw/Jw==";
+
+        private const string CosmosDbDefaultAccountName = 
+            "localhost:8081";
+
+        public CosmosDbTestcontainerConfiguration()
+            : this(CosmosDbImage) { }
+
+        public CosmosDbTestcontainerConfiguration(string image)
+            : base(image, CosmosDbPort, CosmosDbPort)
```

Comment:
I recommend `base(image, CosmosDbPort)`. It is good practice to use a random assigned host port to avoid port clashes.